### ISSUE TITLE
Options page missing?

### ISSUE THREAD

**Issue by MattPurland** [mkdo/link-picker-for-cmb2#4]
I'm looking for the 'Link Picker for CMB2' options page under 'Settings' but it seems to be missing?

**Comment by mattwatsoncodes** [comment]
@MattPurland it doesn't have an options page. What is it you are trying to do?

**Comment by MattPurland** [comment]
You may want to update the readme, under Installation, point 4:

"Configure the plugin via the 'Link Picker for CMB2' options page under the WordPress 'Settings' Menu"

I'd like to include custom taxonomy items (if possible)

**Comment by mattwatsoncodes** [comment]
Ah right @MattPurland, yeah I'll update that. 

You just add the reference to your CMB2 meta box config, so if you have CMB2 on a custom taxonomy, it will work there.

**Comment by mattwatsoncodes** [comment]
@MattPurland Just checked the README.md installation point 4, that just tells you to activate the plugin under the plugin section. It doesn't reference an options page.

**Comment by mattwatsoncodes** [comment]
Ah right, @MattPurland spotted it, in readme.txt. Will fix!

**Comment by MattPurland** [comment]
No problem.

Regarding the custom taxonomy, I'm not entirely sure what you mean. Just to clarify, I have a custom taxonomy called 'Brands' containing 'Brand One', 'Brand Two', etc. I'd like these Brands to be available for link picking. Can you clarify how I 'add the reference'?

**Comment by mattwatsoncodes** [comment]
@MattPurland I'm not sure this is the plugin for you. 

The plugin provides a link picker field for the CMB2 (https://en-gb.wordpress.org/plugins/cmb2/) meta box framework. 

If you integrate it into CMB2 then in the meta box you have created, a WordPress link picker will appear, similar to what appears above the editor.

**Comment by MattPurland** [comment]
Yes I understand that, I'm using it fine throughout my project. But when using a link picker and browsing for links the list of available links doesn't include custom taxonomies, only post types and pages

**Comment by mattwatsoncodes** [comment]
@MattPurland got it. This plugin doesn't do that, it should be handled by WP.

**Comment by MattPurland** [comment]
You're right, I didn't realise you're leveraging the built in link search. My apologies!

**Comment by mattwatsoncodes** [comment]
Reopened issue, missing text is still an issue.

**Comment by mattwatsoncodes** [comment]
Also another issue here: https://wordpress.org/support/topic/missing-options-page/

**Comment by m9n** [comment]
Should be resolved by https://github.com/mwtsn/link-picker-for-cmb2/pull/11
